synapse/storage/data_stores/main/events.py

```diff
@@ -53,6 +54,47 @@
     ["type", "origin_type", "origin_entity"],
 )
 
+STATE_EVENT_TYPES_TO_MARK_UNREAD = {
+    EventTypes.PowerLevels,
+    EventTypes.Topic,
+    EventTypes.Name,
+    EventTypes.RoomAvatar,
+}
+
+
+def count_as_unread(event: EventBase, context: EventContext) -> bool:
```

Comment:
this sounds like an order rather than a question.

```suggestion
def should_count_as_unread(event: EventBase, context: EventContext) -> bool:
```